NotifyMe.js

```diff
@@ -164,7 +164,7 @@ const NotifyMe = props => {
                         <Popover.Title as="h3">{heading}</Popover.Title>
                         <Popover.Content style={{ padding: '3px 3px' }}>
                             {showCount && <div>
-                                <Button variant="link" onClick={markAsRead}>
+                                <Button variant="link" onClick={props.markAsRead || markAsRead}>
                                     <BookOpen size={24} />
                                     Mark all as read
```

Comment:
The bottom of the file has the `props` validations. It should be there as an optional prop and function as type.

Reply:
done